app/views/orga/seasons/_form.html.slim~

```diff
@@ -0,0 +1,16 @@
+= simple_form_for([:orga, @season]) do |f|
```

Comment:
This seems like a backup file, automatically created by your editor. Can you please remove this from the PR again? Hint: it's always best to select/stage the files for a commit individually; avoid using `git add .` which will also add unwanted backup files like this one.